doc/tests/check-nix-code-blocks.nix

```diff
@@ -0,0 +1,27 @@
+{
+  runCommand,
+  markdown-code-runner,
+  nixfmt-rfc-style,
+}:
+
+runCommand "manual_check-nix-code-blocks"
+  {
+    nativeBuildInputs = [
+      markdown-code-runner
+      nixfmt-rfc-style
+    ];
+  }
+  ''
+    set +e
+
+    mdcr --check --config ${./mdcr-config.toml} ${./..}
```

Comment:
Won't this copy the source to the store another time if it's already in the store?

```suggestion
    mdcr --check --config ${./mdcr-config.toml} ${toString ./..}
```

Reply:
I have tried this, it seems that it doesn't work as it should. I don't really understand why

Reply:
What exactly doesn't work here?

Reply:
Test it quickly by doing:

```
nix build github:drupol/nixpkgs/push-pxwnzkyyyvzx#nixpkgs-manual.tests.check-nix-code-blocks -L
```

You can see that it doesn't find any errors.

Reply:
In that case my suggestion wasn't helpful, sorry :)

Reply:
No problem, I restored branch, setting it to review now.